dev-python/pipenv/pipenv-2020.6.2.ebuild

```diff
@@ -33,14 +37,13 @@ DEPEND="test? (
 		dev-python/pytz[${PYTHON_USEDEP}]
 	)"
 
-RESTRICT="!test? ( test )"
-
-S="${WORKDIR}"/${PN}-${MY_PV}
-
-PATCHES=(
-	"${FILESDIR}/${PN}-2020.4.1_beta1-disable-networked-tests.patch"
-)
-
 python_test() {
-	pytest -m "not cli" -vv tests/unit || die
+	pytest -m "not cli and not needs_internet" -vv tests/unit || die
+}
+
+src_prepare() {
+	# remove vendored version of PyYAML that is backported to Python2
+	# this should be removed when upstream removes support for Python2
+	rm -vR "${S}/${PN}/patched/yaml2/" || die
+	eapply_user
```

Comment:
This is wrong, and you have 2 options to fix it:

- Change this to `distutils-r1_src_prepare`
- Rename `src_prepare` to `python_prepare_all` and change this to `distutils-r1_python_prepare_all`

They are basically equivalent.  Regardless of which one you choose, this function should still be above python_test, as mgorny notes.

Reply:
I changed this to `distutils-r1_src_prepare` removed `eapply_user`.

Reply:
Well, I re-added `eapply_user`, since without this I get the following error:
```
* ERROR: dev-python/pipenv-2020.6.2::oz123-overlay failed (prepare phase):
 *   eapply_user (or default) must be called in src_prepare()!
 * 
 * Call stack:
 *            ebuild.sh, line  762:  Called __ebuild_main 'prepare'
 *   phase-functions.sh, line 1050:  Called __dyn_prepare
 *   phase-functions.sh, line  395:  Called die
 * The specific snippet of code:
 *   		die "eapply_user (or default) must be called in src_prepare()!"
 * 
```